easybuild/easyblocks/i/itac.py

```diff
@@ -62,11 +63,18 @@ def install_step(self):
         """
 
         if LooseVersion(self.version) >= LooseVersion('8.1'):
-            super(EB_itac, self).install_step(silent_cfg_names_map=None)
+            silent_cfg_names_map = {}
+            if LooseVersion(self.version) >= LooseVersion('9.0'):
+                # To generate the correct itacvars.sh (called by traceanalyzer) we need to change the default install dir
+                silent_cfg_names_map = {
+                    'install_dir': install_path(),
```

Comment:
I'm not sure that this is the right solution here...

The installation directory is by default named like the module is, so this is OK when your modules are named `<name>/<version>`.

If they're not, because you care installing with a different module naming scheme, `itac` will not get installed where it's supposed to.

With HMNS for example, you'd get `itac/<version>` directly in your installation prefix, while it should be in `Core/itac/<version>`...

So, the proper fix is to actually leave the `move_after_install` in place, and patch the `VT_ROOT=` line in `itacvars.sh`?